product_configurator/models/product.py

```diff
@@ -159,7 +163,7 @@ def _check_default_values(self):
                 value_ids=default_val_ids, product_tmpl_id=self.id, final=False
             )
         except ValidationError as exc:
-            raise ValidationError(exc.args[0]) from exc
+            raise exc
```

Comment:
Should we just delete this `except` block, rather than simply reraising the same exception?

Reply:
This is here probably to raise `ValidationError`s as they are, otherwise they would be caught from the following `except Exception` that just shows a generic error to the user

Reply:
I dislike this, but nothing that can't be improved later, so let's move on.